modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -114,44 +100,120 @@ rule _check_chrFiles:
         touch("references/{genome_build}/freec/chr/.all_done")
 
 
+rule _dbsnp_to_bed:
```

Comment:
Can you please set all rules to adhere to the convention `_controlfreec_..`

Reply:
Okay!